docs/src/lib/representations.md

```diff
@@ -366,6 +367,7 @@ VPolytope
 dim(::VPolytope)
 σ(::AbstractVector{Real}, ::VPolytope{Real})
 vertices_list(::VPolytope)
+intersection(::VPolytope, ::VPolytope)
```

Comment:
Currently we cannot create docs for functions inside the `@require` block because we do not load `Polyhedra`, which again is because that package does not build in v0.7 at the moment.
So you have to remove these two lines again.

Reply:
ok, got it